clustered/server/src/main/java/org/ehcache/clustered/server/offheap/OffHeapChainMap.java

```diff
@@ -167,6 +182,17 @@ public void clear() {
     }
   }
 
+  private void evict() {
+    int evictionIndex = heads.getEvictionIndex();
+    if (evictionIndex < 0) {
+      StringBuilder sb = new StringBuilder("Storage Engine and Eviction Failed - Everything Pinned (" + getSize() + " mappings) \n");
```

Comment:
This winds up allocating two (2) `StringBuilder` instances -- use `append` instead of the concatenation.